src/main/resources/application.properties

```diff
@@ -1,5 +1,14 @@
+# undertow
 server.undertow.accesslog.enabled=false
 server.undertow.accesslog.dir=out/logs
 server.undertow.accesslog.pattern=combined
 server.compression.enabled=false
 server.compression.min-response-size=1
+
+## security
+security.basic.enabled=false
+
+## zalando oauth2
+zalando.oauth2.resource.enabled=true
+zalando.oauth2.client.access-token-uri=https://token.services.auth.zalando.com/oauth2/access_token?realm=/services
```

Comment:
if you're service doesn't act as a client to some other service, then you don't need the zalando.oauth2.client.* key-value pair